Automattic/simperium
# Adds mdb:insert_many support

### Details:
In this PR we're implementing `insert_many` support, capable of inserting a list of documents in any given collection.

### Testing:
1. Launch your `simperium-local-dev` instance (**run.sh** !!)
2. Run `rebar3 shell` in your `simperium` checkout
3. Invoke insert_many as follows:

```
mdb:insert_many(<<"keywords-danger-0fa">>, <<"c.yosemite">>, [ #{ <<"_id">> => <<"first">> }, #{ <<"_id">> => <<"second">> }  ]).
```

**Note:** Please replace `keywords-danger-0fa` with a valid AppID

- [ ] Verify the new documents got inserted


#### Querying Mongo
For self future reference, documenting the steps i've followed below.

```
docker ps | grep mongo
docker exec -it [PID] /bin/bash
mongo
```

Once in there, you may query the `c.yosemite` collection as follows:

```
use keywords-danger-0fa
db.c.yosemite.find()
```
